cmd/goboy/main.go

```diff
@@ -58,6 +55,10 @@ func start() {
 		defer pprof.StopCPUProfile()
 	}
 
+	if *unlocked {
+		*mute = true
+	}
```

Comment:
This sounds like a good change to me until we can get the audio running at the same speed (if we want to that is, you're right in that it probably isn't worth it as it's not nice to listen to!)